pkg/backend/display/diff.go

```diff
@@ -137,10 +137,17 @@ func renderStdoutColorEvent(payload engine.StdoutEventPayload, opts Options) str
 	return opts.Color.Colorize(payload.Message)
 }
 
-func renderSummaryEvent(action apitype.UpdateKind, event engine.SummaryEventPayload, opts Options) string {
+func renderSummaryEvent(action apitype.UpdateKind, event engine.SummaryEventPayload, wroteDiagnosticHeader bool, opts Options) string {
 	changes := event.ResourceChanges
 
 	out := &bytes.Buffer{}
+
+	// If it is a Preview & it failed, we just render the Policy Packs ran. This because rendering the summary for a preview that failed may be
```

Comment:
```suggestion
	// If this is a failed preview, we only render the Policy Packs that ran. This is because rendering the summary for a failed preview may be
```